MoneySplitter.Win10/CustomControls/TransactionEventControl.xaml.cs

```diff
@@ -0,0 +1,52 @@
+﻿using MoneySplitter.Models;
+using MoneySplitter.Win10.CustomControls.Models;
+using System;
+using System.Collections.Generic;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace MoneySplitter.Win10.CustomControls
+{
+	public sealed partial class TransactionEventControl : UserControl
+	{
+		public event EventHandler<TransactionEventModel> OnGiveTransactionButtonClick;
+		public event EventHandler<TransactionEventModel> OnRemindTransactionButtonClick;
+		public event EventHandler<TransactionEventModel> OnApproveTransactionButtonClick;
+
+
```

Comment:
empty spaaaaaaaaaaaaces
what are we living for....

Reply:
delete